package.json

```diff
@@ -22,7 +22,7 @@
     "@skyux/i18n": "^4.0.0 || ^5.0.0-alpha.0",
     "@skyux/indicators": "^4.0.0 || ^5.0.0-alpha.0",
     "@skyux/layout": "^4.0.0 || ^5.0.0-alpha.0",
-    "angular-tree-component": "^8.5.6"
+    "@circlon/angular-tree-component": "^9.0.0"
```

Comment:
Okay, so here's the tricky part: while `@circlon/angular-tree-component@9` works with our build tools (`@skyux-sdk/builder`), our consumers will definitely be using Angular 12. Are there any significant API changes between `@circlon` 9-12? If not, we can probably set this peer range to `"^9.0.0 || ^12.0.0` (is there a 12 version yet?).

Reply:
There is no version 12. The only breaking changes listed between 9 and 11 are:
- lodash removed. getNodeById() has to be updated to no longer use lodash find method.
- Angular v10 minimum dependencies set.

Reply:
Just a note- `@circlon` 11 does enforce v11 minimum dependencies even though the changelog doesn't call it out.

Reply:
I don't believe that we use anything breaking internally. So could we do `>=9.0.0` with a potential ceiling if we wanted?

Reply:
Per our offline discussion, updated angular dependency to `12` and `@circlon/angular-tree-component` to `11`.